Runtime/Tracking/Modification/TransformPropertyApplier.cs

```diff
@@ -298,6 +297,13 @@ protected virtual Vector3 CalculatePositionWithOffset(Vector3 sourcePosition, Ve
         /// <param name="currentPosition">The current position to apply to the target.</param>
         protected virtual void ProcessTransform(TransformData source, TransformData target, Vector3 currentScale, Quaternion currentRotation, Vector3 currentPosition)
         {
+            if (ArePropertiesEqual(target.Position, currentPosition, target.Rotation, currentRotation, target.Scale, currentScale))

```

Comment:
This actually works because the order is correct but it's technically inverted.

The `ArePropertiesEqual` method goes start/destination

`current` is the start and `target` is the destination

so this should read:

`ArePropertiesEqual(currentPosition, target.Position, currentRotation, target.Rotation, currentScale, target.Scale)`